cypress/e2e/Class01.cy.js

```diff
@@ -0,0 +1,69 @@
+/// <reference types="Cypress" />
+
+describe('Validate EngageSphere website', ()=> {
+  beforeEach(()=> {
+    cy.visit('/')
+  })
+  context('Title', ()=> {
+    it('Validate title', ()=> {
+      cy.contains('h1','EngageSphere').should('be.visible')
+    })
+    it('Validate theme icon', ()=>{
+      cy.get('[id="theme-toggle-button"]').should('be.visible')
+    })   
+  })
+  context('Label', ()=>{
```

Comment:
Que tal assim, visto que é a funcionalidade de _greeting_ (saudação) que está sendo testada neste `context`?

```suggestion
  context('Greeting', ()=>{
```